src/Symfony/Component/Finder/Finder.php

```diff
@@ -136,13 +136,104 @@ public function depth($level)
      *
      * @return $this
      *
+     * @deprecated since 4.1, to be removed in 5.0
```

Comment:
now changed to 
> should to be removed in 5.0.

Reply:
Re-reading this PR, I think `date()` should be keep as is. Using the `modified` date is the most useful comparison, so it deserves to be the default.